python/llm/example/transformers_int4.py

```diff
@@ -32,7 +32,7 @@
     if model_path == 'decapoda-research/llama-7b-hf':
         # load_in_4bit=True in bigdl.llm.transformers will convert
         # the relevant layers in the model into int4 format
-        model = AutoModelForCausalLM.from_pretrained(model_path, load_in_4bit=True)
+        model = AutoModelForCausalLM.from_pretrained(model_path, load_in_4bit=True, low_cpu_mem_usage=True)
```

Comment:
do we still need `low_cpu_mem_usage`?

Reply:
Yes, otherwise transformer will use twice as much memory to load the model.

We can automatically set that value to true, but this will require user to install HF `accelerate`. Do you want it to be set automatically?

Reply:
shall we add HF `accelerate` to our dependency? @TheaperDeng @shane-huang

Reply:
@jason-dai 

`accelerate` (about 200k in size) is already in the dependencies. `low_cpu_mem_usage` will be set automatically when user sets `load_in_4bit`